docs/Overview.md

```diff
@@ -35,37 +35,37 @@ and "OPTIONAL" in this documentation set are to be interpreted as described in [
 
 ### Input
 
-An Input is a unit consuming media data for providing it to Sender(s). Input may present metadata about media consuming capabilities to its upstream counterpart. Input is a resource and based on [Resource Core JSON Schema][Resource-Core-Schema].
+An Input is a unit consuming media data for providing it to Sender(s). Input may present metadata about media consuming capabilities to its upstream counterpart. Each Input is a resource and based on [Resource Core JSON Schema][Resource-Core-Schema].
 
 ### Output
 
-An Output is a unit producing media data provided by Receiver(s). Output may hold metadata about the media consuming capabilities from its downstream counterpart. Output is a resource and based on [Resource Core JSON Schema][Resource-Core-Schema].
+An Output is a unit producing media data provided by Receiver(s). Output may hold metadata about the media consuming capabilities from its downstream counterpart. Each Output is a resource and based on [Resource Core JSON Schema][Resource-Core-Schema].
 
 ## API Structure
 
-The API provides a mechanism to get information about Inputs associated with Senders and Outpus associated with Receivers and to constrain Sender from sending Flow not supported by its subscribers.
+The API provides a mechanism to get information about Inputs associated with Senders and Outputs associated with Receivers and to constrain a Sender from sending a Flow not supported by its subscribers.
 
 The API is divided into the following parts:
 
 ### Senders
 
-Each Sender has `inputs` resource, which lists identifiers of all Inputs associated with this Sender, `constraints` base resource in charge of restrictions for the Flow associated with this Sender and `status` resource showing Sender's state in terms of IS-11.
+Each Sender has an `inputs` resource, which lists identifiers of all Inputs associated with this Sender, a `constraints` base resource in charge of restrictions for the Flow associated with this Sender and a `status` resource showing the Sender's state in terms of IS-11.
 
 #### Constraints
 
-There is `active` resource which holds Active Constraints and `supported` resource which lists all the Parameter Constraints supported by this Sender.
+There is an `active` resource which holds Active Constraints and a `supported` resource which lists all the Constraint Sets supported by this Sender.
```

Comment:
Thank you Gareth! The only thing is that a Sender lists supported Parameter Constraints to show which of them make sense for the Sender and it can try to adhere to them.

```suggestion
There is an `active` resource which holds Active Constraints and a `supported` resource which lists all the Parameter Constraints supported by this Sender.
```

Reply:
Ah, sorry, yes!!